HLTrigger/Muon/plugins/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<library   file="*.cc" name="HLTriggerMuonPlugins">
```

Comment:
HLTrigger/Muon already defines only plugins (in the /src directory), you should not use a new /plugins directory, just put the .cc files in the /src directory.